tests/config/pytest_modules.yml

```diff
@@ -2976,14 +2976,6 @@ roary:
   - modules/nf-core/roary/**
   - tests/modules/nf-core/roary/**
 
-rsem/calculateexpression:
-  - modules/nf-core/rsem/calculateexpression/**
-  - tests/modules/nf-core/rsem/calculateexpression/**
```

Comment:
please also remove these old test files

Reply:
@sateeshperi wanted us to keep them for now